CMakeLists.txt

```diff
@@ -257,6 +258,9 @@ if(FLATBUFFERS_BUILD_GRPCTEST)
   if(CMAKE_COMPILER_IS_GNUCXX)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-parameter -Wno-shadow")
   endif()
+  INCLUDE_DIRECTORIES(/your/path/to/grpc/install/include)
+  INCLUDE_DIRECTORIES(/your/path/to/protobuf/include)
+  LINK_DIRECTORIES(/your/path/to/grpc/install/lib)
```

Comment:
Why do we have these here? If they are meant as documentation they should maybe be comments?
Last I worked on this code, such include and link dirs are not necessary if you `make install` gRPC, if I remember correctly.

Reply:
You may be right. I did not `make install` gRPC in the system directory. C++ gRPC docs warn against that saying uninstalling will not be easy. So I did `make prefix= $(pwd)/install install`. I'm using includes and libs off of that. Hence the paths.

Just compiling and installing grpc was not enough. While compile `grpc_test.cpp` I got a compiler error that `common.h` is missing
```
Scanning dependencies of target grpctest
[ 71%] Building CXX object CMakeFiles/grpctest.dir/tests/monster_test.grpc.fb.cc.o
In file included from /home/sutambe/projects/my-oss/grpc/install/include/grpcpp/impl/codegen/proto_utils.h:28:0,
                 from /home/sutambe/projects/my-oss/grpc/install/include/grpc++/impl/codegen/proto_utils.h:26,
                 from /home/sutambe/projects/my-oss/myflatbuffers/tests/monster_test.grpc.fb.h:13,
                 from /home/sutambe/projects/my-oss/myflatbuffers/tests/monster_test.grpc.fb.cc:6:
/home/sutambe/projects/my-oss/grpc/install/include/grpcpp/impl/codegen/config_protobuf.h:25:42: fatal error: google/protobuf/stubs/common.h: No such file or directory
 #include <google/protobuf/stubs/common.h>
                                          ^
compilation terminated.
make[2]: *** [CMakeFiles/grpctest.dir/tests/monster_test.grpc.fb.cc.o] Error 1
make[1]: *** [CMakeFiles/grpctest.dir/all] Error 2
make: *** [all] Error 2
```

It's kind of weird that protobuf headers are needed while compiling flatbuffer tests. The fix could be in a separate PR though.

The `INCLUDE_DIRECTORIES` and `LINK_DIRECTORIES` serve dual purpose: CMake code and comments. I'm fine with proper cmake variables. I was wondering about them myself.